src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -117,19 +33,13 @@ static const DoutPrefixProvider* dpp() {
     } \
   } while (0)
 
-static inline int posix_errortrans(int r)
+using namespace std;
+
+inline int posix_errortrans(int r)
```

Comment:
Certainly not necessary to change this, kind of a drive-by edit as I was reading. But, might as well keep it. Parameter 'r' could be const (I think a const-int parameter here makes sense), but I am ok either way.